Kentico.Kontent.Delivery.Extensions.DependencyInjection/DeliveryClientFactory/IDeliveryClientFactoryBuilder.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using Kentico.Kontent.Delivery.Abstractions;
+using Kentico.Kontent.Delivery.Builders.DeliveryClient;
+using Kentico.Kontent.Delivery.Caching;
+using Kentico.Kontent.Delivery.Configuration;
+
+namespace Kentico.Kontent.Delivery.Builders.DeliveryClientFactory
+{
+    // TODO 312 - XML comments
+    // TODO 312 decide whether this lib is the best for this interface (currently here because of IDeliveryClientBuilder dependency)

```

Comment:
As stated in the comment - in the IDeliveryClientBuilder.AddDeliveryClientCache, there is an argument from Kentico.Kontent.Delivery.Caching -> so this can't be in the delivery package.

Reply:
There might be a way of splitting interface in two - let's sync about it.

Reply:
Let's move the interface to Kontent.Ai.Delivery (done in #347 )